scaladoc/resources/dotty_res/styles/theme/layout/container.css

```diff
@@ -19,6 +19,11 @@ p {
   --header-height: calc(8 * var(--base-spacing));
 }
 
+.site-container img{
+  width: 100%;
```

Comment:
Can we change it to `max-width` to not upscale smaller images?
E.g. here: https://dotty.epfl.ch/blog/2018/11/30/11th-dotty-milestone-release.html